README.markdown

```diff
@@ -1094,6 +1095,97 @@ In larger expressions, optional parentheses can sometimes make code read more cl
 ```swift
 let playerMark = (player == current ? "X" : "O")
 ```
+## Linting
+
+ We use SwiftLint to lint our projects so we guarantee our guidelines are being enforced during development. For more details on how SwiftLint works and how to install it please [check this](https://github.com/realm/SwiftLint).
+ 
```

Comment:
How are we using SwiftLint in our projects? As a brew binary or integrated as a pod? I'll add this info. Also, a TL;DR; of the installation method (`brew install swiftlint` or `pod install`)

Reply:
Done b3de5ec